### ISSUE TITLE
Continue the pretraining of CamemBERT on tagged text

### ISSUE THREAD

**Issue by tagny** [lengocluyen/relation_extraction_textmine25#18]
generate mask task dataset and conyinue the pretraining of Camembert-large
- we might not need to fine-tune the whole BERT stack but just the classification head after that
